control/tests/margin_test.py

```diff
@@ -119,6 +118,8 @@ def test_margin_3input(tsys):
      (([2], [1, 3, 3, 1]), [1.732, 0.], [-0.25, 2.]),
      ((np.array([3, 11, 3]) * 1e-4, [1., -2.7145, 2.4562, -0.7408], .1),
       [1.6235, 0.], [-0.28598, 1.88889]),
+     (([200.0], [1.0, 21.0, 20.0, 0.0]),
+      [4.47213595, 0], [-0.47619048, inf]),
```

Comment:
Warnings do not cause the test to fail, so this case does not provide a regression test. To fix this, add the decorator `@pytest.mark.filterwarnings("error")` as done [in another test in margin_test.py](https://github.com/python-control/python-control/blob/2f59e954830fbb98cdde8ca4743cf171c82d4111/control/tests/margin_test.py#L360).